inc/customizer.php

```diff
@@ -67,6 +67,31 @@ function understrap_theme_slug_sanitize_select( $input, $setting ) {
 
 		}
 
+		// Sticky Header options
+		$wp_customize->add_setting(
+			'understrap_header_position',
+			array(
```

Comment:
This is missing a `sanitize_callback`. You could use this for one: 

https://github.com/WPTRT/code-examples/blob/master/customizer/sanitization-callbacks.php#L23